app/views/prosecution_cases/_raw_response.html.haml

```diff
@@ -0,0 +1,39 @@
+%br
+%br
```

Comment:
Instead of using line breaks, try using a spacing override classes: https://design-system.service.gov.uk/styles/spacing/

Reply:
was just a move of existing content to a partial so a bit of tangent but ok

Reply:
can you post an example as the `!` causes problems in haml it think

Reply:
Because `!` causes issues you have to wrap it within braces `{class: 'govuk-!-margin-top-2'}`

Reply:
done now